### PR TITLE
Pathology bundle update

### PR DESCRIPTION
**Pull request by bhashemian**
### Description
It fixes the log print in inference and update README.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [x] In-line docstrings updated.
- [x] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [x] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [x] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [x] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [x] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [x] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/pathology_tumor_detection/docs/README.md [hunk 1]
```diff
@@ -63,7 +63,8 @@ Inference is performed on WSI in a sliding window manner with specified stride.
 # Model Performance
 
 FROC score is used for evaluating the performance of the model. After inference is done, `evaluate_froc.sh` needs to be run to evaluate FROC score based on predicted probability map (output of inference) and the ground truth tumor masks.
-This model achieve the ~0.92 accuracy on validation patches, and FROC of ~0.72 on the 48 Camelyon testing data that have ground truth annotations available.
+This model achieve the ~0.91 accuracy on validation patches, and FROC of ~0.72 on the 48 Camelyon testing data that have ground truth annotations available.
```
Comment (yiheng-wang-nv) [inline models/pathology_tumor_detection/docs/README.md]: Hi @drbeh , as the metric has changed, should the weights be updated as well? May need to change the `large_files.yml`

Comment (bhashemian) [reply models/pathology_tumor_detection/docs/README.md]: Updated.

**Review by yiheng-wang-nv — APPROVED** [review]

**Comment by Nic-Ma** [comment]
Hi @drbeh ,

Is this PR ready to review now?

Thanks for the quick update.

**Comment by bhashemian** [comment]
Hi @Nic-Ma, yes it is.

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**